### DISCUSSION TITLE
Is it possible to create a settings file and save it?

### DISCUSSION THREAD

**Discussion by noonand** [microsoft/terminal#14780] [General]
Hello all,

I'm wondering if it's possible with Terminal to create a settings file that you can save and pass around to your team. This settings file would have whatever consoles that you need to be defined in it for a particular purpose.

For example, if I'm doing front-end Angular development I might choose to start up the apps that I will need to debug and develop against. For the back-end, I might make sure that SQL Server is started and accepting connections along with whatever queuing software and utilities listening on specific ports that I need to debug and develop against.

This is how some existing tools such as Cmder and ConEmu work and I was wondering if (_hoping that_)  Terminal has similar functionality.

- Derek

**Comment by zadjii-msft** [comment]
Yep, you can find the [settings.json file here](https://github.com/microsoft/terminal/wiki/Frequently-Asked-Questions-(FAQ)#where-can-i-find-the-settings-file). 

You may also be interested in ["fragment extensions"](https://learn.microsoft.com/en-us/windows/terminal/json-fragment-extensions), which would let you author one common json file and drop it in `%localappdata%\Microsoft\Windows Terminal\Fragments\whatever\whatever.json`, so that your teammates can all have their own unique settings files, with the profiles from `whatever.json` just simply added to their settings.
